.rspec-local

```diff
@@ -0,0 +1 @@
+--tag ~redis
```

Comment:
Did you intend to commit this? 

It was added to `.gitignore` in #920, so I assume it’s meant to be local-only.

Reply:
Ah damn well caught. Yep ignored in one branch and committed in the other 😭